src/runmode-af-packet.c

```diff
@@ -292,6 +292,11 @@ static void *ParseAFPConfig(const char *iface)
         }
     }
 
+    if (strcmp(iface, out_iface) == 0) {
```

Comment:
indeed. It segfaults here in that case. Apparently, it can be NULL in the condition above as well so it segfaults there first for a case.
Sending new PR.